iron-menu-behavior.html

```diff
@@ -101,7 +107,11 @@
       var selectedItem = this.multi ? (this.selectedItems && this.selectedItems[0]) : this.selectedItem;
 
       this.items.forEach(function(item) {
-        item.setAttribute('tabindex', item === selectedItem ? '0' : '-1');
+        // Don't enable tab index if an item is disabled, even if it's selected.
+        if (item === selectedItem && !selectedItem.hasAttribute('disabled') && !this.disabled)
+          item.setAttribute('tabindex', 0);
+        else
+          item.removeAttribute('tabindex');
```

Comment:
Done, reverted changes to the items/focusedItem to use -1 tab index because we don't control the type of elements we'll get as items.

However, I kept the change in _disabledChanged because dbeam@ pointed out that it is still clickable in [this CL](https://codereview.chromium.org/2715763002).